manifests/c/Cisco/Webex/Bundle/44.12.0.31404/Cisco.Webex.Bundle.installer.yaml

```
@@ -0,0 +1,43 @@
+# Created with komac v2.8.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Cisco.Webex.Bundle
+PackageVersion: 44.12.0.31404
+InstallerLocale: en-US
+InstallerType: wix
+InstallerSwitches:
+  InstallLocation: INSTALL_ROOT="<INSTALLPATH>"
+  Custom: ACCEPT_EULA=TRUE
+UpgradeBehavior: uninstallPrevious
+Protocols:
+- ciscospark
+- wbx
+- webex
+- webexediscovery
+- webexteams
+FileExtensions:
+- ciscospark
+- wbx
+- webex
+- webexediscovery
+- webexteams
+ProductCode: '{A04D0B8C-CF5F-5B72-8BD5-8A899DB199EE}'
+ReleaseDate: 2024-12-17
+AppsAndFeaturesEntries:
+- DisplayName: Webex
+  Publisher: Cisco Systems, Inc
+  ProductCode: '{A04D0B8C-CF5F-5B72-8BD5-8A899DB199EE}'
+  UpgradeCode: '{B9DCA8E9-B3A9-419A-9D6F-9BC3557EE72C}'
+InstallationMetadata:
+  DefaultInstallLocation: '%ProgramFiles%\Cisco Spark'
+Installers:
+- Architecture: x64
+  Scope: user
+  InstallerUrl: http://asus-gt302/d/WebexBundle.msi
```

```suggestion
  InstallerUrl: https://binaries.webex.com/Webex-Desktop-Windows-x64-Combined-Gold/WebexBundle.msi
```

Uses a local server as installer file cache.

Accidentally committed without switching back to the official URL. 🤣